src/modules/StickyTR.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from "prop-types"
+import React from "react"
+
+export default class StickyTR extends React.Component {
+  renderChildren = (child, columnIndex) => {
+    const width = this.props.widths[columnIndex]
+    return React.cloneElement(child, { width })
```

Comment:
so if I use my own `<th>` here and this clones it, will any props I have be lost or does `{ width }` add to my component's props?

Reply:
https://reactjs.org/docs/react-api.html#cloneelement

Reply:
👍